R/customVarInfo.R

```diff
@@ -44,8 +44,7 @@
   longType = NULL,
   includeAttributes = NULL,
   varInfo = list(),
-  position = 1
-) {
+  position = 1) {
```

Comment:
I personally prefer having the `)` on the same indentation level as the opening `... function(` to have visual separator between the arguments and the body. But if this is best practice or according to some style guide I don't mind the change either... :)

Reply:
Sorry, auto formatting changes this. I'll revert this as you prefer the original.